### PR TITLE
Student/Teacher Views

### PR DESCRIPTION
**Pull request by the-pat**
Modified ClassListDetails to have different views for Lecture and Class Index (StudentIndex and TeacherIndex).

### CODE REVIEW

File: UniWatch/Views/Class/TeacherIndex.cshtml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                    </td>
+                    <td>
+                        @Html.ActionLink("Delete", "Delete", new {classId = item.Id})
+                    </td>
+                </tr>
+            }
+        </table>
+    </div>
+}
+
```
Comment (ClaireGray) [inline UniWatch/Views/Class/TeacherIndex.cshtml:59]: I have look everything patrick. Thanks for fixing some stuff I have overlook. I have approved all changes in here.

File: UniWatch/Views/Class/StudentIndex.cshtml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+@{
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+<h3>My Classes</h3>
+
+@if (!Model.Any())
+{
+    @:It's empty in here. Create a new class.
```
Comment (frankibem) [inline UniWatch/Views/Class/StudentIndex.cshtml]: Copy and paste? A student cannot create a class. Change to something like "It's empty in here. Your classes will show up once a teacher enrolls you in his/her class"

File: UniWatch/Views/Class/StudentIndex.cshtml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                    @Html.DisplayNameFor(model => model.Number)
+                </th>
+                <th>
+                    @Html.DisplayNameFor(model => model.Section)
+                </th>
+                <th>
+                    @Html.DisplayNameFor(model => model.Year)
+                </th>
+                <th>
+                    @Html.DisplayNameFor(model => model.Teacher)
```
Comment (frankibem) [inline UniWatch/Views/Class/StudentIndex.cshtml]: Change to "Teacher"

File: UniWatch/Views/Class/StudentIndex.cshtml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+                        @Html.DisplayFor(modelItem => item.Number)
+                    </td>
+                    <td>
+                        @Html.DisplayFor(modelItem => item.Section)
+                    </td>
+                    <td>
+                        @Html.DisplayFor(modelItem => item.Year)
+                    </td>
+                    <td>
+                        @Html.DisplayFor(modelItem => item.Teacher.FirstName)
```
Comment (frankibem) [inline UniWatch/Views/Class/StudentIndex.cshtml]: Use string.Format() to combine

**Review by frankibem — CHANGES_REQUESTED** [review]

**Review by frankibem — APPROVED** [review]

**Pull request merged**